Documentation/project-docs/standard-platform.md

```diff
@@ -0,0 +1,362 @@
+
+Today, Portable Class Libraries (PCL) target an intersection of APIs depending on your platform selection when making the project. This gives you a specific surface area that guarantees you work on the chosen platforms. Those combinations are precomputed to give you the right set of surface area. When these portable libraries are packaged into NuGet, they are expressed with a static set of frameworks e.g. **portable-net45+win8**. While this describes the intent that you want to run on .NET Framework 4.5 and Windows 8.0, it is also restrictive, since new platforms can appear in the future that are perfectly capable of running those PCLs but are blocked due to the platforms that were selected when the project was created. In fact, putting the portable dll inside of a folder with a static list of profiles *essentially* makes it platform-specific. It's no different to doing:
+
+```
+MyLibrary/net45/MyLibrary.dll
+MyLibrary/win8/MyLibrary.dll
+```
+
+The biggest difference is that you wouldn't be able to consume it in a Portable Class Library project type.
+
+The .NET Standard Platform version represents binary portability across platforms using a **single** moniker. They are an evolution of the existing Portable Class Library system. They are "open-ended" in that they aren't tied down to a static list of monikers like **portable-a+b+c** is.
+
+.NET Standard Platform versions are not too different to the PortableXXX profiles people use today which get represented in NuGet as **portable-a+b+c** (eg. Profile111). The key difference is that the **single** .NET Standard Platform moniker evolves and versions linearly, such that NuGet and other tools can infer compatibility, i.e. newer .NET Standard Platform versions are compatible with older ones.
+
+
+## Terms
+- **PCL** - Portable Class Library
+- **Platform** - e.g. .NET Framework 4.5, Windows Phone 8.1
+- **Reference Assembly** - An assembly that contains API surface only. There is no IL in the method bodies. It is used for compilation only, and cannot be used to run. Also commonly referred to as "Contracts".
+- **Implementation Assembly** - An assembly that contains an implementation of a reference assembly. These are usually implemented in the platform itself and cannot be updated without updating the platform.
+- **.NET Standard Platform** - versioned sets of the reference assemblies.
+- **Cross compile** - to compile the same source code files to different target platforms, i.e. against different API sets.
+
+## Principles
+- Platform owners implement reference assemblies from a particular .NET Standard Platform version.
+- Platform owners may implement a subset of reference assemblies from a particular .NET Standard Platform version.
+- Any change in a reference assembly's API surface causes the .NET Standard Platform to version.
+- Lower versions are always compatible with higher versions.
```

Comment:
Rule 2 is about platform owners implementing the Standard Platform.  But for the Standard Platform itself, we'd never remove anything in a newer version?  And even with 2, we'd never remove anything from a contract?  Just trying to clarify.